lectures/ifp_egm.md

```diff
@@ -102,32 +101,35 @@ The following function stores default parameter values for the income
 fluctuation problem and creates suitable arrays.
 
 ```{code-cell} ipython3
-def ifp(R=1.01,             # gross interest rate
-        β=0.99,             # discount factor
-        γ=1.5,              # CRRA preference parameter
-        s_max=16,           # savings grid max
-        s_size=200,         # savings grid size
-        ρ=0.99,             # income persistence
-        ν=0.02,             # income volatility
-        y_size=25):         # income grid size
-  
+Household = namedtuple('Household', ('β', 'R', 'γ', 's_size', 'y_size', \
+                                     's_grid', 'y_grid', 'P'))
+
+def create_household(R=1.01,             # gross interest rate
+                     β=0.99,             # discount factor
+                     γ=1.5,              # CRRA preference parameter
+                     s_max=16,           # savings grid max
+                     s_size=200,         # savings grid size
+                     ρ=0.99,             # income persistence
+                     ν=0.02,             # income volatility
+                     y_size=25):         # income grid size
```

Comment:
Let's migrate the comments to where the `namedtuple` is defined.

Reply:
I agree --- in fact let's have both, because the arguments are not identical to the elements of the namedtuple.  Some overlap is fine.